tile_generator/package_flags.py

```diff
@@ -503,14 +505,15 @@ def _apply(self, config_obj, package, release):
                             'token': '(( .properties.k8s_cluster_token.value ))',
                             'service_id': '(( .properties.service_id.value ))',
                             'helm_chart_dir': '/var/vcap/packages/charts_for_%s/chart' % packagename,
+                            'operator_dir': '/var/vcap/packages/charts_for_%s/operator_chart' % packagename,
                         }
                     },
                 }, {
                     'name': 'loader',
                     'dynamic_ip': 1,
                     'post_deploy': True,
                     'lifecycle': 'errand',
-                    'ephemeral_disk': package.get('ephemeral_disk', get_disk_size_for_chart(package['helm_chart_dir']) * 3),
+                    'ephemeral_disk': package.get('ephemeral_disk', get_disk_size_for_chart(package['helm_chart_dir'], package['operator_dir'] if 'operator_dir' in package else None) * 3),
```

Comment:
See comment above about pythonic way of using `.get('operator_dir')`

Reply:
Done.